### ISSUE TITLE
Expense notifications

### ISSUE THREAD

**Issue by asood123** [opencollective/opencollective#281]
Currently: a mess! I can't even describe it.

Going forward:
1. When an expense is filed, we will:
- [x] generate a new activity:`GROUP_EXPENSE_CREATED` (same as today)
- [x] send a slack and/or gitter notifications to the appropriate group (if they have subscribed)
- [x] email group `MEMBERS` with the info, picture of receipt and `APPROVE` and `REJECT` button

2. When an expense is approved, we will:
- [x] generate a new activity: `GROUP_EXPENSE_APPROVED` (same as today)
- [x] send a slack and gitter ... (same as 1)
- [x] email group `HOST` asking them to pay expense (note: currently not supported via email, instead we'll send them a link to the expense on the website)
- [ ] [Nice to have] email group `MEMBER`s and the original expense filer as an FYI

3. When an expense is rejected, we will:
- [x] generate a new activity: `GROUP_EXPENSE_REJECTED` (same as today)
- [x] send a slack and gitter... (same as 2)
- [ ] email group `MEMBERS`... (same as 2)

4. When an expense is paid, we will:
- [x] generate a new activity: `GROUP_EXPENSE_PAID` (new)
- [x] send a slack and gitter... (same as 2a)
- [x] email group `MEMBER`s and expense filer confirming that expense is paid
- [ ] add info in the email above to include which email address was paid, how much is remaining, etc.

Let me know if anything else is needed for expense notifications.

**Comment by xdamman** [comment]
This looks good. The only thing I'd say is that you shouldn't send the notifications to all members but to all users who subscribed to that notification. That way they can unsubscribe.

I also think that it would be good to add some text in plain English in those notification emails to explain what's the process and what to expect next. Eg: "your expense is now approved. The host (BrusselsTogether ASBL) should reimburse you shortly on your xxx@xxx.com Paypal email"

**Comment by asood123** [comment]
Yep, goes without saying that all email notifications are subscription based.

Plaintext is a good idea - added some of it already, let's keep developing it further once the end to end system is running.
